catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -236,13 +246,12 @@
 		<argument ref="federatedSources"/>
 		<argument ref="resourceReaderSortedList"/>
 		<argument ref="sorted"/>
```

Comment:
Can use the same bean here, otherwise the thread pool will be shared with `CachingFederationStrategy`.